src/overlays/actors/ovl_En_Zl1/z_en_zl1.c

```diff
@@ -27,39 +37,626 @@ const ActorInit En_Zl1_InitVars = {
     (ActorFunc)EnZl1_Update,
     (ActorFunc)EnZl1_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Zl1/func_80B4AB40.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Zl1/EnZl1_Init.s")
+ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK0, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x01, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 20, 46, 0, { 0, 0, 0 } },
+};
+
+UNK_TYPE D_80B4E61C[] = { 0x06007208, 0x06009848, 0x06009C48, 0x06009848 };
+UNK_TYPE D_80B4E62C[] = { 0x06007608 };
+
+typedef struct {
+    u8 unk_00[0x3];
+} u8_3;
+
+typedef struct {
+    u8 unk_00[0xA];
+} u8_A;
+
+typedef struct {
+    AnimationHeader* unk_00[0xA];
+} AH_A;
+
+Vec3f D_80B4E630 = { -460.0f, 118.0f, 0.0f };
+Vec3f D_80B4E63C = { -406.0f, 110.0f, 0.0f };
+Vec3f D_80B4E648 = { -398.0f, 84.0f, 0.0f };
+Vec3f D_80B4E654 = { -427.0f, 108.0, 26.0 };
+Vec3f D_80B4E660 = { -340.0f, 108.0f, 98.0f };
+Vec3f D_80B4E66C = { -434.0f, 84.0f, 0.0f };
+
+u8_3 D_80B4E678 = { 0, 0, 2 };
+
+AH_A D_80B4E67C = {
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x06012B04, 0x06012118, 0x06010B38,
+};
+
+u8_A D_80B4E6A4 = {
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00, 0x02,
+};
+
+Vec3f D_80B4E6B0 = { -421.0f, 143.0f, -5.0f };
+Vec3f D_80B4E6BC = { -512.0f, 105.0f, -4.0f };
+Vec3f D_80B4E6C8 = { 0.0f, 0.0f, 0.0f };
+
+AH_A D_80B4E6D4 = {
+    0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
+    0x00000000, 0x00000000, 0x06012B04, 0x06012118, 0x06010B38,
+};
+
+u8_A D_80B4E6FC = {
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00, 0x02,
+};
+
+Vec3f D_80B4E708 = { 0.0f, 0.0f, 0.0f };
+Vec3f D_80B4E714 = { 0.0f, 0.0f, 0.0f };
+
+extern AnimationHeader D_06000438;
+extern u32 D_06008848;
+extern u32 D_06008C48;
+extern SkeletonHeader D_0600F5D8;
+extern AnimationHeader D_06010B38;
+extern AnimationHeader D_06011348;
+extern AnimationHeader D_060116E4;
+extern AnimationHeader D_06011B88;
+extern AnimationHeader D_06012118;
+extern AnimationHeader D_060132D8;
+extern AnimationHeader D_060138E0;
+extern AnimationHeader D_06013F10;
+extern AnimationHeader D_060143A8;
+
+void func_80B4AB40(){
+};
+
+void func_80B4AB48(){
+};
```

Comment:
Style isn't quite right here, not sure if it's just missed by clang-format or what. Also `(void)` is better than `()` for functions that don't take arguments.